ui/app/styles/core/title.scss

```diff
@@ -2,6 +2,10 @@
 .subtitle:not(:last-child) {
   display: block;
   margin-bottom: 1rem;
+
```

Comment:
My original reason for using important was because title was overriding the margin-bottom class. So I added this here. It's a duplication of the has-bottom-margin-xs, but it doesn't use important. @hashishaw thoughts?

Reply:
👍 I think it's better than using !important 😄 thanks!